zebra-state/src/service/check/issuance.rs

```diff
@@ -50,6 +56,13 @@ pub fn valid_burns_and_issuance(
             }
         }
 
+        // FIXME: Not sure: it looks like semantically_verified.issued_assets_changes is already
+        // filled with burn and issuance items in zebra-consensus, see Verifier::call function in
+        // zebra-consensus/src/transaction.rs (it uses from_burn and from_issue_action AssetStateChange
+        // methods from ebra-chain/src/orchard_zsa/asset_state.rs). Can't it cause a duplication?
+        // Can we collect all change items here, not in zebra-consensus (and so we don't need
+        // SemanticallyVerifiedBlock::issued_assets_changes at all), or performing part of the
+        // checks in zebra-consensus is important for the consensus checks order in a some way?
```

Comment:
> Can't it cause a duplication?

It wouldn't be duplicated since the previous loop isn't mutating the asset states, it's just loading them into memory.

> Can we collect all change items here, not in zebra-consensus

Yep, I'll try to make this change.